src/core/runtime.d

```diff
@@ -227,27 +227,8 @@ struct Runtime
         import core.stdc.stdlib : free, malloc;
         version (Windows)
         {
-            import core.sys.windows.windows;
-
-            if (name.length == 0) return null;
-            // Load a DLL at runtime
-            auto len = MultiByteToWideChar(
-                CP_UTF8, 0, name.ptr, cast(int)name.length, null, 0);
-            if (len == 0)
-                return null;
-
-            auto buf = cast(WCHAR*)malloc((len+1) * WCHAR.sizeof);
-            if (buf is null) return null;
-            scope (exit) free(buf);
-
-            len = MultiByteToWideChar(
-                CP_UTF8, 0, name.ptr, cast(int)name.length, buf, len);
-            if (len == 0)
-                return null;
-
-            buf[len] = '\0';
-
-            return rt_loadLibraryW(buf);
+            name[$ - 1] = '\0';
```

Comment:
Is this actually tested?  Given `in char[] name` I would expect mutation to fail.

Reply:
Yes, fails to compile. Time to add a test similar to test/uuid?